### PR TITLE
Fix ms groups list members

### PR DESCRIPTION
**Pull request by YuvHayun**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [XSUP-21528](https://jira-hq.paloaltonetworks.local/browse/XSUP-21528)

## Description
Added the **count** argument to **msgraph-groups-list-members** command. to fix the issue above

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

**Comment by JudahSchwartz** [comment]
@YuvHayun can you please explain how this fixed the issue? How did you learn this was what was missing?

**Comment by YuvHayun** [comment]
> @YuvHayun can you please explain how this fixed the issue? How did you learn this was what was missing?

I reproduced the issue locally and then added the customer's suggestion that actually fixed it.
Then I also found [this](https://learn.microsoft.com/en-us/graph/aad-advanced-queries?tabs=http) article about advanced queries that require the header and count.

File: Packs/MicrosoftGraphGroups/Integrations/MicrosoftGraphGroups/MicrosoftGraphGroups.yml [hunk 1]
```diff
@@ -390,6 +390,15 @@ script:
...
       secret: false
+    - default: false
+      description: Retrieves the total count of matching resources.
+      isArray: false
+      name: count
+      required: false
+      secret: false
+      auto: PREDEFINED
+      predefined:
+      - 'true'
```
Comment (Shellyber) [inline Packs/MicrosoftGraphGroups/Integrations/MicrosoftGraphGroups/MicrosoftGraphGroups.yml:401]: Shouldn't this be true and false? as options?

**Review by Shellyber — COMMENTED** [review]
@YuvHayun Can you add a test?

File: Packs/MicrosoftGraphGroups/Integrations/MicrosoftGraphGroups/MicrosoftGraphGroups.yml (hunk 1 above)
Comment (YuvHayun) [reply Packs/MicrosoftGraphGroups/Integrations/MicrosoftGraphGroups/MicrosoftGraphGroups.yml:401]: I could only find information about using the argument with 'true'
Also, when I tried to run it with 'false' I encountered an exception.

**Review by Shellyber — APPROVED** [review]
Nice job

**Pull request merged**